interacting-dom/app.js

```diff
@@ -0,0 +1,52 @@
+const btnCreate = document.getElementById("btn-main");
+const btnToggle = document.querySelector(".btn-toggle");
+const btnRemove = document.querySelector(".btn-remove");
+const taskList = document.querySelector(".list-container ul");
+const remove = document.querySelectorAll('li');
```

Comment:
I would prefer more descriptive names than remove. It seems like that could be anything and isn't really descriptive of a list item..